packages/canary-bot/src/server.ts

```diff
@@ -0,0 +1,26 @@
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import {GCFBootstrapper} from 'gcf-utils';
+import appFn from './canary-bot';
+
+const bootstrap = new GCFBootstrapper();
+const server = bootstrap.server(appFn);
+const port = 8080;
```

Comment:
Cloud Run provides port for us, which could be different between instances - perhaps we should use `process.env.PORT` while falling back to `8080`:

```ts
const port = process.env.PORT ?? 8080;
```